src/__tests__/taskList.test.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import TaskList from '../modules/taskList.js';
+
+const todo = new TaskList();
+
+describe('Testing add functionality', () => {
+  test('Add todo', () => {
+    todo.addTodo('Have fun', todo.tasks);
+    expect(todo.tasks[0].desp).toEqual('Have fun');
+  });
+});
+
```

Comment:
Please test the functions `#clearCompleted` and `#clickCheck` and use mock for functions that use `localStorage`. Kindly make sure all the methods for adding and removing the items are tested.

Reply:
Sorry, but from my understanding, these are not among the requirement for part one of this project.
Please find the requirement link here: https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/tests_1.md